pkg/hyper/helper.go

```diff
@@ -271,3 +292,53 @@ func toKubeContainerState(state string) kubeapi.ContainerState {
 		return kubeapi.ContainerState_CONTAINER_UNKNOWN
 	}
 }
+
+// TODO(harry) These two methods will find subsystem mount point frequently, consider move FindCgroupMountpoint into a unified place.
+// getCpuLimitFromCgroup get the cpu limit from given cgroupParent
+func getCpuLimitFromCgroup(cgroupParent string) (int32, error) {
+	mntPath, err := libcontainercgroups.FindCgroupMountpoint("cpu")
+	if err != nil {
+		return -1, err
+	}
+	contents, err := ioutil.ReadFile(filepath.Join(mntPath, cgroupParent, cpuQuotaCgroupFile))
+	if err != nil {
+		return -1, err
+	}
+
+	cpuQuotaStr := strings.TrimSpace(string(contents))
+
+	if value, err := strconv.Atoi(cpuQuotaStr); err == nil {
+		// k8s use 100 milicpu to describe cpu quota
+		vcpuNumber := value / cpu100m
```

Comment:
I think `cpu100m` should be got from `cpu.cfs_period_us` and it's better to use round calculation for vcpu numbers (so 1800m should get two vcpus instead of one).